bigtable/use-cases/fraudDetection/src/test/java/FraudDetectionTestUtil.java

```diff
@@ -60,25 +62,27 @@ private static void parseTerraformOutput(Process terraformProcess) throws IOExce
     String line;
     while ((line = reader.readLine()) != null) {
       System.out.println(line);
```

Comment:
Can you just do a return at the start checking if the line contains UNKOWN VALUE rather than checking in each place?